rego/rego_test.go

```diff
@@ -1735,7 +1736,12 @@ func TestPrepareWithWasmTargetNotSupported(t *testing.T) {
 	})
 }
 
+
 func TestPrepareAndEvalWithWasmTarget(t *testing.T) {
```

Comment:
I guess we could put these into an extra file, with a buildflag, too...?

Reply:
Yeah that's probably cleaner! I'll try that 👍